test/unit.h

```diff
@@ -136,6 +141,110 @@ int check_plan(void);
 	}						\
 }
 
+#if UNIT_TAP_COMPATIBLE
```

Comment:
Leaving `UNIT_TAP_COMPATIBLE` solely because of `rb.c` is unacceptable. Please rewrite `rb.c` so that it's TAP compatible as well and drop this macro, like it was done [here](https://github.com/tarantool/msgpuck/commit/aa159e0e0fe80fe9dd5dca686183bf28f0bd5880), for example. It should be pretty easy - the test basically prints 1, 2, 3, 4 ... and then in reverse so you could check it with `is` in the callback.

Reply:
Done. Dropped `UNIT_TAP_COMPATIBLE`. `rb.c` is made TAP compatible too.